BlueprintUI/Sources/Layout/GeometryReader.swift

```diff
@@ -48,8 +48,8 @@ public struct GeometryProxy {
     /// The size constraint of the element being laid out.
     public var constraint: SizeConstraint
 
-    /// Measure the given element, using this proxy's constraint and the current environment.
-    public func measure(element: Element) -> CGSize {
-        element.content.measure(in: constraint, environment: environment)
+    /// Measure the given element, using the provided constraint (or the proxy's constraint if not provided), and the current environment.
+    public func measure(in explicit: SizeConstraint? = nil, element: Element) -> CGSize {
```

Comment:
Ah true; I had avoided putting the element first since we usually avoid that on `init` methods for container elements, but its unlikely you'll be constructing a new element here so it wont leave the orphan "in:foo" likely.

Reply:
Done